javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  let i = 0;
+  while (i < values.length) {
+    if (typeof values[i] === 'string') {
+      values.splice(i, 1);
+    } else {
+      i++;
+    }
+  }
+  return values;
+}
```

Comment:
Same note as above ^ about manipulating the array directly.

Reply:
Updated. Thank you for letting me know.